keyboards/thevankeyboards/caravan/readme.md

```diff
@@ -0,0 +1,15 @@
+# Caravan
+
+![Caravan](https://imgur.com/a/6hxhN0r)
+
+A compact split 44% keyboard.
+
+Keyboard Maintainer: QMK Community
+Hardware Supported: Caravan PCB
```

Comment:
```suggestion
Hardware Supported: Caravan PCB  
```

These two lines need the two trailing spaces, in order for GitHub to render them correctly.